_overviews/getting-started/scala-ides.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: singlepage-overview
+title: Scala IDEs
+
+partof: scala-ides
+
+permalink: /tutorials/:title.html
```

Comment:
this permalink is the source of the HTMLproofer errors - the url is `/tutorials/scala-ides.html` - but it should probably be 
```suggestion
permalink: /getting-started/:title.html
```